### DISCUSSION TITLE
Akku Entladesteuerung

### DISCUSSION THREAD

**Discussion by JauntyJosef** [wiggal/GEN24_Ladesteuerung#62] [Q&A]
Ich muss leider schon wieder etwas fragen, sorry.

Wenn ich bei "Feste Entladegrenze" "0.1" eingebe, dann wird das in der `Akku_EntLadeSteuerFile.json` mit dem Wert 100 eingetragen. Das scheint ja soweit richtig zu sein.

Im Log steht dann allerdings 0% als Batterieentladegrenze und es wird tatsächlich nicht entladen.

```
Feste Entladegrenze: 100 %
Batteriestatus in Prozent: 64.3 %
GesamtverbrauchHaus: 348 W
VerbrauchsgrenzeEntladung: 0 W
Batterieentladegrenze ALT: 100 %
Batterieentladegrenze NEU: 0 %
```
Mit zB 1 kw als Entladegrenze passts dann.

Habe ich wieder etwas übersehen?

**Comment by wiggal** [comment]
Hi,
Fragen zu stellen ist überhaubt kein Problem, wenn ich keine Zeit habe bekommst du halt keine Antwort ;-)

Der Efekt bei 0.1KW entsteht durch folgende Formel:
`int(FesteEntladegrenze/BattganzeLadeKapazWatt*100)`
100W/11000*100 = 0.909 
als Ganze Zahl ist das dann halt 0.
Bei 0.2 müsste dann schon ein Wert raus kommen.

Schönen Gruß

**Reply by JauntyJosef** [reply]
Ah, ok. Das macht Sinn. Da stimmt das in der Hilfe nicht ganz, denn da steht min 0.1. Das würde dann bei einem Akku über 10kWh nicht mehr funktionieren.
Die `BattganzeLadeKapazWatt` wird aus dem Modbusregister oder aus er config.ini `MaxLadung` gelesen?

**Reply by wiggal** [reply]
Ja es kann nur minimal 0.1 gesetzt werden, bei größeren Akkus  kann man dann die Entladung nicht auf Null setzen, war für die Grundlast gedacht, da bin ich davon ausgegangen, dass die nicht Null wird.

BattganzeLadeKapazWatt kommt aus dem Modbusregister (bei mir 11059W)

**Reply by JauntyJosef** [reply]
Alles klar. 100W sind ja eh eher unrealistisch in der Realität, ich würde es aber eventuell in der Hilfe bei einer zukünftigen Version ändern.
Danke sehr.
